TrafficCapture/trafficReplayer/src/main/resources/log4j2.properties

```diff
@@ -1,5 +1,7 @@
 status = error
 
+property.tupleDir = /shared-replayer-output/${env:TUPLE_DIR_NAME}
```

Comment:
Would we be able to make this whole thing an environment variable, with a default, so that it works on developer boxes?

Reply:
Great idea, I have adjusted so there is a default here if the env variable is not found